webpack-subresource-integrity-embroider/index.js

```diff
@@ -31,6 +32,14 @@ class SubresourceIntegrityPlugin {
               return;
             }
 
+            if (fileName.startsWith("http")) {
```

Comment:
We should test for protocol-relative URLs starting with (`//`) as well.

```suggestion
            if (fileName.startsWith("http") || fileName.startsWith("//")) {
```